memory/memory.ts

```diff
@@ -34,13 +34,20 @@ interface Session {
 export class Memory implements Session, MemorySession {
   store: URL;
   ready: Promise<unknown>;
+  private _serviceDid: DID;
+
   constructor(
     options: Options,
     public subscribers: Set<Subscriber> = new Set(),
     public spaces: Map<Subject, SpaceSession> = new Map(),
   ) {
     this.store = options.store;
     this.ready = Promise.resolve();
+    this._serviceDid = options.serviceDid;
+  }
+
+  serviceDid(): DID {
+    return this._serviceDid;
```

Comment:
```suggestion
  ready: Promise<unknown>;
  #serviceDid: DID;

  constructor(
    options: Options,
    public subscribers: Set<Subscriber> = new Set(),
    public spaces: Map<Subject, SpaceSession> = new Map(),
  ) {
    this.store = options.store;
    this.ready = Promise.resolve();
    this.#serviceDid = options.serviceDid;
  }

  serviceDid(): DID {
    return this.#serviceDid;
```

Reply:
Unless there is a good reason, I would suggest using actual JS privates as opposed to TS pretend privates.